src/XrdHttp/XrdHttpReq.cc

```diff
@@ -1188,8 +1191,7 @@ int XrdHttpReq::ProcessHTTPReq() {
 
             if (!prot->Bridge->Run((char *) &xrdreq, 0, 0)) {
               mapXrdErrorToHttpStatus();
-              sendFooterError("Could not run close request on the bridge");
-              return -1;
+              return sendFooterError("Could not run close request on the bridge");
```

Comment:
I think it we already discussed that it's better not to hide the return value in `sendFooterError`, but return the error explicitly afterwards (which also allow returning something different than `-1` if needed.

Reply:
I'm not sure we discussed it (apologies if I just forgot).

What are you suggesting?  Doing something like:

```
if (sendFooterError("blah") == -1) {
    return -1;
} else {
    return 1;
}
```

This seems redundant so I must be missing some aspect of the suggestion.

If we ever want to make it so `sendFooterError` returns different values than `ProcessHTTPReq` we can certainly refactor then.

Reply:
@amadio - doing a pass through all my open PRs.  Kind reminder about the above question.  I'm not sure I understand your original question so I don't quite know how to address your concern.

Thanks!

Reply:
The previous discussion was in #2300 when I [requested changes](https://github.com/xrootd/xrootd/pull/2300#pullrequestreview-2404575191). In particular, I had requested to make the function void here: https://github.com/xrootd/xrootd/pull/2300/files#r1822452730

Reply:
Ok, we have discussed in person and agree that it's easier to just let this function return something.